desktop.css

```diff
@@ -0,0 +1,377 @@
+  @import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+
+  * {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+
+  body,
+  html {
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+  }
+
+  main {
+    width: 100%;
+    height: 4561px;
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .desktop-navbar-container {
+    display: block;
+    width: 100%;
+    height: 72px;
+    background-color: #3e3c3c;
+  }
+
+  .desktop-navbar {
+    display: flex;
+    gap: 8px;
+    padding: 1.5% 18% 0 10%;
+  }
+
+  .autor {
+    font-family: Inter, sans-serif;
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 20px;
+    letter-spacing: 0;
+    text-align: center;
+    width: 92px;
+    height: 20px;
+    margin-right: auto;
+  }
+
+  .desk-port {
+    width: 63px;
+    height: 20px;
+  }
+
+  .desk-about {
+    width: 45px;
+    height: 20px;
+  }
+
+  .desk-contact {
+    width: 59px;
+    height: 20px;
+  }
+
+  .desk-contact,
+  .desk-about,
+  .desk-port {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .desktop-navbar a {
+    color: #fff;
+  }
+
+  .mailbox {
+    width: 24px;
+    height: 24px;
+    cursor: pointer;
+  }
+
+  /* Hero styles */
+  .sec-container {
+    width: 100%;
+    height: auto;
+    background-image: url("./desktop-img/Illustration-header.svg");
+    background-repeat: no-repeat;
+    background-size: 1003.13px 769.27px;
+    background-position-x: right;
+    background-position-y: top;
+  }
+
+  .hero {
+    display: flex;
+    flex-direction: row;
+    gap: 50px;
+  }
+
+  .desk-hero-text {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 192px 0;
+    gap: 24px;
+    width: 100%;
+    height: 739px;
+    margin-right: 51%;
+  }
+
+  .header {
+    width: 100%;
+    height: 160px;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    padding-left: 0;
+  }
+
+  .sub-header {
+    width: 100%;
+    height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    padding-left: 0%;
+    margin-left: 0;
+  }
+
+  .para {
+    width: 100%;
+    height: 123px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    padding-left: 0;
+  }
+
+  .anchor {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    order: -1;
+    width: 20px;
+    height: auto;
+    margin-left: 2%;
+  }
+
+  /* work section */
+
+  .main-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 130px 24px;
+    width: 100%;
+    height: auto;
+  }
```

Comment:
- [ ] You did a great job implementing the hover effect on the regular project cards, but the projects section is not responsive and on some small desktop screen sizes, the page cuts the elements. Kindly make it responsive so all the cards can appear on the screen even on small desktop sections.
 - Your design:
 - 
<img width="1121" alt="image" src="https://user-images.githubusercontent.com/61919617/222478802-0bd60fc2-2b64-4c7a-a2a3-2dcee9327acf.png">

 - Figma template:
 - 
<img width="1114" alt="image" src="https://user-images.githubusercontent.com/61919617/222479354-b6f5980f-bd69-4587-9e80-2943b2e5cdcd.png">